backend/models/listingModel.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const listingSchema = new Schema({
+  organisationName: {
+    type: String,
+    required: true
+  },
+  title: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    required: true
+  },
+  address: {
+    firstLine: {
+      type: String,
+      required: true
+    },
+    city: {
+      type: String,
+      required: true
+    },
+    postcode: {
```

Comment:
Needs some validation added - length and postcode pattern (using regex?). Important as we're going to be using this field for mapping